jdk/src/share/bin/jli_util.h

```diff
@@ -101,4 +107,32 @@ void     JLI_TraceLauncher(const char* fmt, ...);
 void     JLI_SetTraceLauncher();
 jboolean JLI_IsTraceLauncher();
 
+/*
+ * JLI_List - a dynamic list of char*
+ */
+struct JLI_List_
+{
+    char **elements;
+    size_t size;
+    size_t capacity;
+};
+typedef struct JLI_List_ *JLI_List;
+
+JLI_List
+JLI_List_new(size_t capacity);
```

Comment:
Perhaps this declaration, and the ones beginning on lines 128 and 135, should be on one line like the others in this file.

Reply:
Updated.